raiden/network/rpc/smartcontract_proxy.py

```diff
@@ -140,8 +140,8 @@ def transact(
     def get_transaction_data(
             abi: Dict,
             function_name: str,
-            args: List = None,
-            kwargs: Dict = None,
+            args: Any = None,
+            kwargs: Any = None,
     ):
         """Get encoded transaction data"""
         args = args or list()
```

Comment:
Comparing `args: Any`, `kwargs: Any` to the change in https://github.com/raiden-network/raiden/pull/3927/files#diff-79ef917c2473698777a5474a23dc119fR674 (`constructor_parameters: Tuple[Any] = None` and consecutive assignment to a new variable `ctor_parameters`) makes me wonder, if we could come up with a consistent style for optional argument assignments?

Reply:
If i understand you correctly, so you are suggesting to choose either one of:
1. `X or ()`
2. `X or list()`
?